GitUI/AutoCompletion/CommitAutoCompleteProvider.cs

```diff
@@ -98,10 +98,9 @@ private static IEnumerable<string> ReadOrInitializeAutoCompleteRegexes()
             Validates.NotNull(appDataPath);
 
             var path = PathUtil.Combine(appDataPath, "AutoCompleteRegexes.txt");
-
             if (File.Exists(path))
             {
-                return File.ReadLines(path);
+                return File.ReadLines(path!);
```

Comment:
I'm surprised this `!` is needed. I'd think `File.Exists` would be annotated such that a true return would imply `path` is non-null.